### PR TITLE
feat(cli): add prerelease tag

### PR DESCRIPTION
**Pull request by lecoqjacob**
Addresses https://github.com/spicylobstergames/SpicyLauncher/issues/19

image

### CODE REVIEW

File: cli/src/app.rs [hunk 1]
```diff
@@ -81,8 +85,15 @@ impl App {
                     String::from("not installed")
                 }
             );
+
+            if release.prerelease {
+                println!("{release_title} [{}]", "prerelease".red().to_string());
```
Comment (orhun) [inline cli/src/app.rs:90]: Maybe `orange` color would be better since `red` is usually a sign of something negative/erroneous. What do you think?

**Comment by orhun** [comment]
CI is failing. Can you run `cargo clippy` and apply the suggestions?

File: cli/src/app.rs (hunk 1 above)
Comment (lecoqjacob) [reply cli/src/app.rs:90]: yeah i agree. I wanted the user to be cautious that the build was prerelase and red stood out the most to me. Let me play with some colors and then post it here

**Comment by lecoqjacob** [comment]
@orhun 

magenta
image

red:
image

yellow:
image

purple:
image

cyan
image

**Comment by zicklag** [comment]
Yellow seems the most natural to me, and matches GitHub:

image

My only hesitation is that the title is also yellow, but I think yellow's probably still the best color.

**Comment by lecoqjacob** [comment]
Is the name of the Release color able to change too? Seems off to have the same color per line

**Comment by orhun** [comment]
Yeah, feel free to play with the release color too. I agree with @zicklag, it would fit better if we have yellow for the prereleases. 

> **Edit:** Maybe it should also be `pre-release` instead of `prerelease`.

Indeed 👍🏼

**Comment by lecoqjacob** [comment]
@orhun  Went with the green layout for release name :)

**Review by orhun — APPROVED** [review]
Thanks!

**Pull request merged**